layouts/jobs/bis.html

```diff
@@ -0,0 +1,6 @@
+{{ define "main" }}
+    {{ range .Params.bis }}
+        <h2><a href="{{ .link }}">{{ .name }}</a></h2>
+        {{ .description | markdownify }}
```

Comment:
Could potentially user `RenderString` instead of `markdownify` here if we thought we might need custom markdown rendering in the future for this.